src/components/sidenav/sidenav.ts

```diff
@@ -325,6 +325,7 @@ export class MdSidenavLayout implements AfterContentInit {
     }
   }
 
+  /* tslint:disable:no-unused-variable */
```

Comment:
tslint doesn't exclude private class members in the no-unused-varaible rule

![screen shot 2016-03-08 at 1 14 56 pm](https://cloud.githubusercontent.com/assets/8420490/13613012/dfe5cb22-e52f-11e5-86df-34734a69f5c7.png)